pkg/mcs/resource_manager/server/config.go

```diff
@@ -32,13 +32,15 @@ import (
 
 const (
 	defaultName              = "Resource Manager"
-	defaultBackendEndpoints  = "127.0.0.1:2379"
-	defaultListenAddr        = "127.0.0.1:3380"
+	defaultBackendEndpoints  = "http://127.0.0.1:2379"
+	defaultListenAddr        = "127.0.0.1:2380"
```

Comment:
Should we keep consistent with `defaultBackendEndpoints`, e.g., using the prefix? And defaultListenAddr is the resource manager's address, it's better to be different from the peer URL of etcd.